### PR TITLE
Added R scripts

### PR DESCRIPTION
**Pull request by gabtam55**

### CODE REVIEW

File: Practice/Shiny_App.R [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+# Build UI----------------------------------------------------------------------
+ui <- fluidPage(
+  
+  titlePanel("Mental Illness & Socioeconomic Status"),
+  
+  fluidRow(
+    column(5, offset = 1,
+           sliderTextInput("income", "Household Income",
+                           choices = c("$0-$9,999", "$10,000-$24,999", "$25,000-$49,999", "$50,000-$74,999", "$75,000-$99,999",
+                                       "$100,000-$124,999", "$125,000-$149,999", "$150,000-$174,999", "$175,000-$199,999", "$200,000+"),
```
Comment (avisionh) [inline Practice/Shiny_App.R:17]: In response to your question:
> *How can I take the input from line 15-18 in the server to filter 'survey_data' based on the Household Income selected?*
```suggestion
                           choices = survey_data$`Household Income`,
```

Comment (avisionh) [reply Practice/Shiny_App.R:17]: Not sure if this is what you're looking for?

File: Practice/Shiny_App.R [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+
+# Build UI----------------------------------------------------------------------
+ui <- fluidPage(
+  
+  titlePanel("Mental Illness & Socioeconomic Status"),
+  
+  fluidRow(
+    column(5, offset = 1,
+           sliderTextInput("income", "Household Income",
```
Comment (avisionh) [inline Practice/Shiny_App.R:15]: In response to your question:
> *How can I take the input from line 15-18 in the server to filter 'survey_data' based on the Household Income selected?*

This is where naming the arguments for functions come handy. In the below, the `inputId = income` so we will pass this into the `server <- ..` part of the script as `filter(`Household Income` == input$income)`
```suggestion
           sliderTextInput(inputId = "income", label = "Household Income",
```

File: Practice/Shiny_App.R [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+    )
+  )
+)
+
+# Build Server------------------------------------------------------------------
+server <- function(input, output, session) {
+  
+  filtered_data <- reactive({
+    survey_data %>%
+      filter()
```
Comment (avisionh) [inline Practice/Shiny_App.R:72]: In response to your question:
> *How can I take the input from line 15-18 in the server to filter 'survey_data' based on the Household Income selected?*

```suggestion
      filter(`Household Income` == input$income)
```

**Review by avisionh — COMMENTED** [review]
Happy to provide further feedback on this if you want.

File: Practice/Shiny_App.R (hunk 3 above)
Comment (gabtam55) [reply Practice/Shiny_App.R:72]: Thanks Avision. It helps for my first step! If you are interested, I have put the complete code here - https://github.com/gabtam55/Practices/blob/master/190820%20-%20Mental%20Health%20%26%20Socioeconomic%20Status/App.R

**Pull request closed without merging**